lib/sequelize.js

```diff
@@ -200,9 +200,9 @@ class Sequelize {
     }
```

Comment:
Line 198, which looks like a complex empty check can be removed now,  just having this should work

```js
password: config.password || this.options.password || null
```

Reply:
I removed the empty check of the password.